devtools/manifest.json

```diff
@@ -18,6 +18,7 @@
   "permissions": ["<all_urls>", "activeTab", "webNavigation"],
   "web_accessible_resources" : [
     "shared/arc-registry.js",
+    "shared/invoke-planner.js",
```

Comment:
maybe shared/*.js instead of the two files listed explicitly? Though in the other comment I'm suggesting moving the file to runtime/debug, so depends on what will happen here.